src/Conversion/ONNXToTOSA/Math/Gemm.cpp

```diff
@@ -165,6 +177,7 @@ class ONNXGemmOpLoweringToTOSA : public OpConversionPattern<ONNXGemmOp> {
     auto AType = A.getType().cast<TensorType>();
     auto BType = B.getType().cast<TensorType>();
 
+    bool isCPresent = !C.getType().isa<mlir::NoneType>();
     // If C is present, it can only be of rank 1
```

Comment:
Slightly confused here; shouldn't we check for `getRank() == 1` according to the comment?

Reply:
Ahhh alright, I see your point now, sure we can check that too. If you find it confusing, there is a high chance that someone else also will